docs/install.md

```diff
@@ -0,0 +1,82 @@
+# Installing and configuration
+
+## Installation
+
+The pulp-cli package can be installed from a variety of sources. After installing, see the next
+section on how to configure pulp-cli.
+
+### From PyPI
+
+```
+pip install pulp-cli[pygments]  # colorized output
+pip install pulp-cli  # no color output
+```
+
+### From a source checkout
+
+```
+git clone <your_fork_url>
+cd pulp-cli
+pip install -e .
+```
+
+## Configuration
+
+The CLI can be configured by using a toml file.
+By default the location of this file is `~/.config/pulp/settings.toml`.
+However, this can be customized by using the `--config` option.
+Any settings supplied as options to a command will override these settings.
+
+Example file:
+
+```
+[cli]
+base_url = "https://pulp.dev"
+verify_ssl = false
+format = "json"
+```
+
+### Generate
+
+To generate a new configuration:
+
+```
+pulp config create --base_url "http://localhost"
+pulp config create -e  # with an editor (e.g. vim)
+pulp config create -i  # interactive prompts
+```
+
+### Validate
+
+To validate an existing config:
+
+```
+pulp config validate
+pulp config validate --strict  # validates that all settings are present
+```
+
+### netrc
+
+If no user/pass is supplied either in the config file or as an option,
+then the CLI will attempt to use `~/.netrc`.
+Here is a `.netrc` example for localhost:
+
+```
+machine localhost
+login admin
+password password
+```
+
+### Katello
+
+If you have a Katello environment and wish to use pulp-cli to connect to Pulp, you'll need to
+configure client certificate authentication:
```

Comment:
Not for this PR, but are we sure, that providing the cert will not collide with an existing netrc config?

Reply:
Yes, I will investigate.

Reply:
I filed an issue so I wouldn't forget https://github.com/pulp/pulp-cli/issues/135